terraform/projects/app-ecs-services/alertmanager-service.tf

```diff
@@ -79,6 +98,8 @@ data "template_file" "alertmanager_container_defn" {
     log_group        = "${aws_cloudwatch_log_group.task_logs.name}"
     region           = "${var.aws_region}"
     config_bucket    = "${aws_s3_bucket.config_bucket.id}"
+    alertmanager_url = "--web.external-url=https://${local.alertmanager_public_fqdns[count.index]}"
```

Comment:
We are defining `alertmanager_url` twice here. If you see my comment below I think we should stick with the first definition and remove this second one.